src/Eto.Gtk/Eto.Gtk.csproj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DefineConstants>TRACE;PCL;DEBUG;NETSTANDARD2_0;GTK3</DefineConstants>
+    <OutputPath>..\..\artifacts\core\Debug\</OutputPath>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DefineConstants>TRACE;PCL;RELEASE;NETSTANDARD2_0;GTK3</DefineConstants>
+    <OutputPath>..\..\artifacts\core\Release\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GtkSharp" Version="3.22.24.28" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Eto\Eto.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="EtoWebView.cs" />
+    <Compile Remove="Forms\Controls\WebViewHandler.cs" />
```

Comment:
Ah, boo.  Any idea if this can be done in the future?  A lot of projects use WebView.

Reply:
That's the Gtk 2 webview, there is separate Gtk 3 webview.